src/components/organisms/UiHorizontalPaging/UiHorizontalPaging.spec.ts

```diff
@@ -26,64 +19,4 @@ describe('HorizontalPaging.vue', () => {
     const wrapper = mount(UiHorizontalPaging);
     expect(wrapper.classes('ui-horizontal-paging')).toBe(true);
   });
```

Comment:
Question: no more to test?

Reply:
No. CL test is outdated. We need to write new tests, which are easiest to maintain.